atomix/cluster/src/main/java/io/atomix/raft/roles/PassiveRole.java

```diff
@@ -237,12 +238,21 @@ private void addSnapshotListener() {
       }
     }
 
-    boolean snapshotChunkConsumptionFailed;
+    boolean snapshotChunkConsumptionFailed = true;
     try {
       snapshotChunkConsumptionFailed = !pendingSnapshot.apply(snapshotChunk).join();
+    } catch (final IOException e) {
+      log.warn(
+          "Failed to write pending snapshot chunk {}, rolling back snapshot {}",
```

Comment:
Maybe we can make it a constant string (static field)

Reply:
What's the advantage here? I typically only make them constants if they're reused or if they're long enough that it's hard to read, but I also didn't give it this much thought.